app/src/main/res/values/strings.xml

```diff
@@ -251,49 +251,49 @@
     <string name="count_milliseconds_short">%d ms</string>
     <string name="use_recommended_fonts">Apply the Recommended Font</string>
     <string name="use_recommended_fonts_notice">Use the supplied custom font</string>
-    <string name="waiting">Waiting</string>
-    <string name="theme_optimize">Optimize for AMOLED Screens</string>
-    <string name="theme_optimize_summary">Use darker colors</string>
+    <string name="waiting">Waiting…</string>
+    <string name="theme_optimize">AMOLED Optimization</string>
+    <string name="theme_optimize_summary">Use less power with darker colors</string>
     <string name="error_not_trusted_notice">You need to be trusted by the client</string>
     <string name="web_share">Web Share</string>
-    <string name="visit_using_browser_notice">Visit this link with your browser</string>
-    <string name="empty_connections_list">Networks you connect appear here</string>
+    <string name="visit_using_browser_notice">Visit this link with your web browser</string>
+    <string name="empty_connections_list">Networks you connect to appear here</string>
     <string name="vpn_interface">VPN Server</string>
     <string name="unrecognized_qr_code_notice">Unrecognized code</string>
     <string name="save_text_success">Text saved</string>
     <string name="show_as_qr_code">Show as QR code</string>
     <string name="delete_received_files_question">Also delete received files from disk</string>
     <string name="clients">Clients</string>
     <string name="crash_report">Crash Report</string>
-    <string name="crash_report_notice">TrebleShot experienced a crash in a previous session.</string>
-    <string name="error_unknown_host">Could not resolve this host address!</string>
+    <string name="crash_report_notice">TrebleShot crashed previously.</string>
+    <string name="error_unknown_host">Could not resolve this host address.</string>
     <string name="ui_theme">Theme</string>
     <string name="dark_theme">Dark</string>
     <string name="light_theme">Light</string>
-    <string name="follow_system_theme">Follows System Theme</string>
+    <string name="follow_system_theme">Follow system</string>
     <string name="battery_saver_theme">Decided by Battery Saver</string>
     <string name="split_apk">Split APK</string>
-    <string name="delete_empty_text_question">Would you like to delete the empty text?</string>
+    <string name="delete_empty_text_question">Delete the empty text?</string>
     <string name="compose">Compose</string>
     <string name="transfer_history">Transfer History</string>
     <string name="choose_client">Choose Client</string>
     <string name="show_all">Show All</string>
     <string name="online_clients">Online Clients</string>
     <string name="add_new_client">Add New Client</string>
     <string name="generate_qr_code">Generate QR Code</string>
-    <string name="nothing_worked_rhetorical">Nothing Worked?</string>
```

Comment:
I am aware that Android design guidelines suggest capitalizing only the first letter of titles and such, however, we are not following that. 

The first letter of each word in titles and button texts should be capitalized, e.g., `Sharing on Web`. 

I will not be commenting on the similar changes below, so please consider what we come to decide here will apply to them as well.

Reply:
Always nice when they get something right.
I wasn't trying to change things, but thought that this qualifies as a full sentence.